model/eventHandler/ResourceUpdateHandler.php

```diff
@@ -42,12 +44,28 @@ public function catchResourceUpdated(ResourceMovedEvent $event): void
             $movedResource->getUri()
         ]);
 
+        if ($movedResource->isClass()) {
+            foreach ($movedResource->getInstances(true) as $item) {
+                $itemUri = $item->getUri();
+                $itemPrivilegesMap[$itemUri] =  $this->getRolePrivilegeRetriever()->retrieveByResourceIds([$itemUri]);
```

Comment:
Shall we remove it from the foreach to avoid calling container multiple times?

```suggestion
                $itemPrivilegesMap[$itemUri] =  $rolePrivilegeRetriever->retrieveByResourceIds([$itemUri]);
```